core/job/handler/v1beta1/job.go

```diff
@@ -137,6 +138,37 @@ func (jh *JobHandler) DeleteJobSpecification(ctx context.Context, deleteRequest
 	}, nil
 }
 
+func (jh *JobHandler) ChangeJobNamespace(ctx context.Context, changeRequest *pb.ChangeJobNamespaceRequest) (*pb.ChangeJobNamespaceResponse, error) {
+	jobSourceTenant, err := tenant.NewTenant(changeRequest.ProjectName, changeRequest.NamespaceName)
+	if err != nil {
+		errorMsg := "failed to adapt source tenant when changing job namespace"
+		jh.l.Error(fmt.Sprintf("%s: %s", errorMsg, err.Error()))
+		return nil, errors.GRPCErr(errors.InvalidArgument(job.EntityJob, "invalid Namespace name"), errorMsg)
```

Comment:
Can use the same error coming from validation failure.

Reply:
i wanted to raise a invalid argument instead of  500 error

Reply:
cool